README.rst

```diff
@@ -180,6 +180,13 @@ Consequently, all ``pandas_td`` code should keep running correctly with
 `here <https://github.com/treasure-data/pytd/issues/new>`__ if you
 noticed any incompatible behaviors.
 
+.. note:: There is a known difference to ``pandas_td.to_td`` function for type conversion.
+   Since :class:`pytd.writer.BulkImportWriter`, default writer pytd, uses CSV as an intermediate file before
+   uploading a table, column type may change via ``pandas.read_csv``. To respect column type as much as possible,
+   you need to pass `"fmt"="msgpack"` argument to ``to_td`` function.
```

Comment:
``fmt="msgpack"``; double quotes for `fmt` are unneeded.

Reply:
good catch